src/link/abstractlink.h

```diff
@@ -225,7 +225,7 @@ class AbstractLink : public QObject
     const AbstractLink& operator=(const AbstractLink& other);
 
     Q_PROPERTY(qint64 byteSize READ byteSize NOTIFY byteSizeChanged)
-    Q_PROPERTY(LinkConfiguration* configuration READ configuration CONSTANT)
+    Q_PROPERTY(const LinkConfiguration* configuration READ configuration CONSTANT)
```

Comment:
I believe that this is breaking PingStatus widget:
![image](https://user-images.githubusercontent.com/1215497/62010257-de77e980-b13e-11e9-9247-711b47f2314a.png)

Reply:
hm.